src/components/pages/observation/list/views/stats/download-as-png.tsx

```diff
@@ -0,0 +1,31 @@
+export default function DownloadAsPng({ ro, h, svgData }) {
```

Comment:
is there a better way to implement this? instead of using the document object and using plain javascript? is there any better utility available in react or nextjs?